### PR TITLE
Feat: generate location photos in search results page

### PR DESCRIPTION
**Pull request by timwangmusic**
## Description
Generate images for locations and display in search result pages.

## Solution
* Add an endpoint to generate signed URL for AI generated location images.
* Add a default AI generated image for LA.
* Set maximum number of pictures per location (default to 5)

## Testing
- [x] Integration testing on Heroku staging
- [x] Added new unit tests

## Checks
- [x] Have you removed commented code?
- [x] Have you used gofmt to format your code?

### CODE REVIEW

File: aws/client.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
+package aws
+
```
Comment (sananand007) [inline aws/client.go:2]: what is this file for ? Is it for accessing the s3 bucket ?

File: aws/client.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	cfg, err := config.LoadDefaultConfig(context.TODO())
+	if err != nil {
+		return nil, err
+	}
+
+	return &Client{
+		s3: s3.NewFromConfig(cfg),
+	}, nil
+}
+
```
Comment (sananand007) [inline aws/client.go:34]: Can you tell a bit about the `3` functions here. ? 
upload , download , presigned

File: planner/openai.go [hunk 3]
```diff
@@ -28,3 +29,29 @@ func chatCompletion(ctx context.Context, msg string) (string, error) {
 
 	return resp.Choices[0].Message.Content, nil
 }
```
Comment (sananand007) [inline planner/openai.go:31]: This is likely the part that uses the `openai` api to generate images ?

**Comment by sananand007** [comment]
Can you describe the PR a bit, I think what you are doing is 
- generating the image during the search 
- storing the image in aws as a link that can be shown in the page 
- why is Redis being used ? is it for storing the image as a cache, so that it can be re-used

**Review by sananand007 — APPROVED** [review]
great job

File: aws/client.go (hunk 1 above)
Comment (timwangmusic) [reply aws/client.go:2]: yes, it's a wrapper for all the AWS service clients. Currently we only have S3 client.

File: assets/js/search_results.js [hunk 4]
```diff
@@ -39,6 +39,32 @@ async function postUserFeedback(planIdx) {
     .catch((err) => console.error(err));
 }
 
+async function getImageForLocation() {
+  console.log("calling image generation function...");
```
Comment (sananand007) [inline assets/js/search_results.js:43]: this is intended to be there (the console.log) ?

File: iowrappers/blobs.go [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package iowrappers
+
+import (
+	"context"
+	"fmt"
+	"github.com/weihesdlegend/Vacation-planner/POI"
+)
+
```
Comment (sananand007) [inline iowrappers/blobs.go:8]: `discuss` This we can go over during the quick meeting today

File: planner/blob.go [hunk 6]
```diff
@@ -0,0 +1,115 @@
+package planner
+
```
Comment (sananand007) [inline planner/blob.go:2]: `discuss` : we can quickly discuss the changes within planner as well

**Review by sananand007 — APPROVED** [review]
lgtm. Let's have a quick meeting to discuss a bit

**Pull request merged**